.github/workflows/cargo-build-lint-and-test.yml

```diff
@@ -21,11 +21,19 @@ on:
     branches: [ "main" ]
 
 env:
+  # General.
   CARGO_TERM_COLOR: always
   RUSTFLAGS: -C instrument-coverage -C strip=symbols -D warnings
   RUSTDOCFLAGS: -D warnings
   LLVM_PROFILE_FILE: modelardb-%p-%m.profraw
 
+  # Minio.
+  AWS_ACCESS_KEY_ID: minioadmin
+  AWS_SECRET_ACCESS_KEY: minioadmin
+  AWS_DEFAULT_REGION: us-west-1
```

Comment:
I don't think it does anything but maybe change to `eu-central-1` for consistency with the Docker Compose file.

Reply:
Fixed.